src/xpcc/architecture/platform/driver/gpio/stm32/gpio.hpp.in

```diff
@@ -355,18 +380,27 @@ public:
 			EXTI->FTSR |=  (mask);
 			break;
 		}
+		%%endif
 	}
 	xpcc_always_inline static bool
 	getExternalInterruptFlag()
 	{
+		%%if target is stm32l4
+		return (EXTI->PR1 & mask);
+		%%else
 		return (EXTI->PR & mask);
+		%%endif
 	}
 	/**	\brief	Reset the interrupt flag in the interrupt routine. */
 	xpcc_always_inline static void
 	acknowledgeExternalInterruptFlag()
 	{
 		// Flags are cleared by writing a one to the flag position.
+		%%if target is stm32l4
+		EXTI->PR1 |= mask;
```

Comment:
No. You know the magic: PAn and PBn are mapped to line n which is mapped to PR1[n]. Hence the `mask`.

PR2 accommodates event lines 35-39 which are assigned to the Peripheral Voltage Monitor (PVM1).